source/form.html

```diff
@@ -0,0 +1,112 @@
+  <meta charset="utf-8">
+  <title>Каталог</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="nav">
+      <a class="nav__logo" href="#">
+        <img src="img/images/logo-1.svg" width="70" height="59" alt="логотип Cat Energy">
+        <img src="img/images/logo-2.svg" width="101" height="18" alt="cat energy">
+      </a>
+      <button class="nav__button-header" type="button"><span class="visually-hidden">Открыть меню</span></button>
+    </nav>
+  </header>
+  <main class="main-form">
+    <section class="main-form__program-form program-form">
+      <form class="program-form__form" action="#" method="post" target="_blank">
+        <h1 class="program-form__title">Подбор программы</h1>
+        <p class="program-form__text">Заполните анкету, и мы подберем
+          программу питания для вашего кота</p>
+        <fieldset class="program-form__blank blank">
+          <legend class="visually-hidden blank__title">Персональные данные</legend>
+          <ul class="program-form__form-list">
+            <li class="program-form__form-item">
+              <label for="name">ИМЯ:*</label>
+              <input type="text" name="name" placeholder="Барсик" id="name" required>
+            </li>
+            <li class="program-form__form-item">
+              <label for="weight">Вес (кг):*</label>
+              <input type="number" name="weight" placeholder="7" id="weight" required>
+            </li>
+            <li class="program-form__form-item">
+              <label for="age">Возраст (лет):</label>
+              <input type="number" name="age" placeholder="7" id="age" required>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="program-form__radio-buttons radio-buttons">
+          <li class="program-form__radio-item">
+            <label for="slimming">Похудение</label>
+            <input type="radio" id="slimming" name="slimming" value="slimming" checked>
+          </li>
+          <li class="program-form__radio-item">
+            <label for="mass-gain">Набор массы</label>
+            <input type="radio" id="mass-gain" name="mass-gain" value="mass-gain">
+          </li>
+          <li class="program-form__radio-item">
+            <label for="advice">Не знаю (Нужен ваш совет)</label>
+            <input type="radio" id="advice" name="advice" value="advice">
+          </li>
+        </fieldset>
+
+        <fieldset class="program-form__contacts-user contacts-user">
+          <legend class="contacts-user__title">Контактные данные<br>(владельца кота)</legend>
+
+          <label for="email">E-mail:*</label>
+          <input type="email" name="mail" placeholder="kuklachev@gmail.com" id="email" required>
+
+          <label for="phone">Телефон:*</label>
+          <input type="phone" name="phone" placeholder="8 (960) 900-60-90" id="phone" required>
+
+        </fieldset>
+
+        <fieldset class="program-form__comment comment">
+          <lable for="comment">Комментарий</lable>
```

Comment:
Учитывая, что у тебя везде `legend`, то и тут лучш так сделать. Подпись потом скрытую добавишь просто.

```suggestion
          <legend>Комментарий</legend>
```